Description: XTerm and WMPrefs debian configuration. Based on Marcelo E. Magallon <mmagallo@debian.org> diff patch.
Author: Rodolfo García Peñas (kix) <kix@kix.es>
Last-Update: 2012-01-09
--- a/WindowMaker/Defaults/WMState.in
+++ b/WindowMaker/Defaults/WMState.in
@@ -10,14 +10,14 @@
         DropCommand = "wmsetbg -u -t %d";
       },
       {
-	Command = xterm;
-	Name = xterm.XTerm;
+	Command = "x-terminal-emulator";
+	Name = "x-terminal-emulator.XTerm";
 	AutoLaunch = No;
 	Forced = No;
 	Position = "0,1";
       },
       {
-	Command = "#wprefs#";
+	Command = "/usr/bin/WPrefs";
 	Name = groupLeader.WPrefs;
 	AutoLaunch = No;
 	Forced = No;
